MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using MobileManager.Models.Devices;
+using MobileManager.Models.Devices.Interfaces;
+using MobileManager.SeleniumConfigs.DataSets.Interfaces;
+using DotLiquid;
+
+namespace MobileManager.SeleniumConfigs.DataSets
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// AndroidSeleniumConfig.
+    /// </summary>
+    public class AndroidSeleniumConfig : Drop, ISeleniumConfig
+    {
+        /// <summary>
+        /// Initializes a new instance of the AndroidSeleniumConfig class.
+        /// </summary>
+        /// <param name="device">IDevice Device</param>
+        public AndroidSeleniumConfig(IDevice device) { 
+            this.Id = device.Id;
+            this.Name = device.Name;
+            this.AppiumEndpoint = device.AppiumEndpoint;
+            this.Host = !string.IsNullOrEmpty(device.AppiumEndpoint) ? new Uri(device.AppiumEndpoint).Host : "";
+            this.Port = !string.IsNullOrEmpty(device.AppiumEndpoint) ? new Uri(device.AppiumEndpoint).Port.ToString() : "";
+        }
+
+        /// <inheritdoc />
+        /// <summary>
+        /// Gets or sets the Id.
+        /// </summary>
+        /// <value>The Id.</value>
+        public string Id { get; set; }

```

Comment:
If you don't set properties anywhere (like only in constructor), you can omit the set to prohibit outside manipulation - same for ios config

Reply:
Make sense in this case